### ISSUE TITLE
Does not follow Laravel's coding style

### ISSUE THREAD

**Issue by Mulkave** [brodkinca/BCA-Laravel-Inspect#8]
With code like this:

``` php
class MyClass
{
    public function someFunction()
    {
        if ($condition == true) {

        } else {

        }
    }
}
```

where it should warn me about
- brace next to class declaration
- if opening brace on new line
- else opening brace on new line

As declared in [Laravel's coding style guide](http://laravel.com/docs/4.2/contributions#coding-style)

**Comment by brodkin** [comment]
At the time that this project was created, around the first v4 beta, I don't think Laravel had an official coding style. In fact, I don't think that it fully followed PSR-1 for that matter. This would definitely be a worthwhile update but I'm not sure when I'll get to it. I'd accept a PR to include these rules however.
It may be worth noting that PHPCS' current version is a release candidate that I've heard will use a different ruleset format so this may want to wait for the full 2.0 release.

**Comment by Mulkave** [comment]
Right, been trying to find ready sniffs specific for Laravel but got no luck, I'm also in a tight sched right now and can't spare time for this as well, so I guess we'll wait for 2.0!
